project-base/assets/js/bin/transDump.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const processTransDump = require('../commands/translations/processDump');
+const defaultDirWithJsFiles = './assets/js/';
+
+const dirsWithJsFiles = process.argv.slice(2).map(parameter => {
+    const keyValueParameter = parameter.split('=');
+    if (keyValueParameter[0] === 'source-dir') {
+        return keyValueParameter[1];
+    }
+}).filter(item => item !== undefined);
+
+if (dirsWithJsFiles.length === 0) {
+    dirsWithJsFiles.push(defaultDirWithJsFiles + '**/*.js');
+}
+
```

Comment:
The (almost) same code is in trans.js. Does it make sense to extract into a separate helper function?

Reply:
d492c98d79ea489dc00c94f8dc2b05ea3c13ebfa